mpmath/tests/test_linalg.py

```diff
@@ -198,6 +198,14 @@ def test_cholesky():
     x = fp.cholesky_solve(A9, b9)
     assert fp.norm(fp.residual(A9, x, b9), fp.inf) == 0
 
+    np = pytest.importorskip("numpy")
+    a = mp.randmatrix(4, 4) + mp.randmatrix(4, 4) * 1j
```

Comment:
Please use a deterministic test instead, like above.  And please avoid numpy dependence here.